client/config/peerhost.go

```diff
@@ -213,6 +216,18 @@ func (p *DaemonOption) Validate() error {
 		}
 	}
 
+	if os.FileMode(p.WorkHomeMode) == 0 {
+		return errors.New("invalid value for WorkHomeMode")
```

Comment:
If not set, use default mode for compatibility

Reply:
Ok that sounds good. Since that check is already being done where these variables are used, I will remove these checks